api/client/model_custom_predictor.go

```diff
@@ -0,0 +1,17 @@
+/*
+ * Merlin
+ *
+ * API Guide for accessing Merlin's model management, deployment, and serving functionalities
+ *
+ * API version: 0.7.0
+ * Generated by: Swagger Codegen (https://github.com/swagger-api/swagger-codegen.git)
+ */
+
+package client
+
+type CustomPredictor struct {
+	Image           string `json:"image,omitempty"`
+	Command         string `json:"command,omitempty"`
+	Args            string `json:"args,omitempty"`
+	IsArtifactExist bool   `json:"is_artifact_exist,omitempty"`
```

Comment:
Can this be hidden from user `IsArtifactExist` ?

Reply:
It seems redundant with `ArtifactUri`

Reply:
the purpose is different, `ArtifactUri` always there when we create new version, but it doesn't mean that `ArtifactUri` is exist in GCS. That's why I add new field `IsArtifactExist` to determine whether users are trying to upload their artifact to mlflow or not